PowerPointLabs/PowerPointLabs/PasteLab/PasteLabMain.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Windows;
+
+using PowerPointLabs.Utils;
+
+using PowerPoint = Microsoft.Office.Interop.PowerPoint;
+
+namespace PowerPointLabs.PasteLab
+{
+    public class PasteLabMain
+    {
+        public static void PasteToFillSlide(Models.PowerPointSlide slide, float width, float height)
+        {
+            if (IsClipboardEmpty())
+            {
+                return;
+            }
+
+            PowerPoint.ShapeRange pastedObject = slide.Shapes.Paste();
+            for (int i = 1; i <= pastedObject.Count; i++)
+            {
+                var shape = new PPShape(pastedObject[i]);
+                shape.AbsoluteHeight = height;
+                shape.AbsoluteWidth = width;
+                shape.VisualTop = 0;
+                shape.VisualLeft = 0;
+            }
+        }
+
+        internal static bool IsClipboardEmpty()
+        {
+            return Clipboard.GetDataObject() == null;
```

Comment:
e.g. can you mock its return value? null, or the desired object. does it make the test stable? because if test is to run in parallel, other features can use `copy()` and this may fail your lab's logic.

to me, `Clipboard.GetDataObject()` is just another input, then why not make it as a parameter explicitly.